src/objects/Line.js

```diff
@@ -13,7 +13,7 @@ const _inverseMatrix = new Matrix4();
 const _ray = new Ray();
 const _sphere = new Sphere();
 
-function Line( geometry, material, mode ) {
+function Line( geometry = new BufferGeometry(), material = new LineBasicMaterial(), mode ) {
```

Comment:
I think it's okay to finally remove the `mode` parameter.

@mrdoob What do you think? Otherwise the signature looks a bit strange again.

Reply:
Sounds good. I'll remove it.